python/hsfs/engine/hive.py

```diff
@@ -340,6 +339,15 @@ def save_stream_dataframe(
             "Stream ingestion is not available on Python environments, because it requires Spark as engine."
         )
 
+    def get_empty_appended_dataframe(self, dataframe, new_features):
```

Comment:
Did you try to run it with the patch to enforce the feature group schema in the ingestion job? If it works, then we don't need this method.